version.go

```diff
@@ -0,0 +1,8 @@
+package rdns
+
+var (
+	BuildVersion string = "3.0.0"
```

Comment:
Do you have a way to generate this file during the build? Just wondering why it's at `3.0.0` already.

Reply:
I was going to add that in a different PR because of using GitHub actions. It is currently 3.0.0 because the version was not set and it was not 1.0.0.

Reply:
I can tag a proper release when I'm back next week. Shall we say `v0.1.0` since it's the first one?

Reply:
Sure